source/internal/_isTypedArray.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Tests whether or not an object is a typed array.
+ *
+ * @private
+ * @param {*} val The object to test.
+ * @return {Boolean} `true` if `val` is a typed array, `false` otherwise.
+ * @example
+ *
+ *      _isTypedArray(new Uint8Array([])); //=> true
+ *      _isTypedArray(new Float32Array([])); //=> true
+ *      _isTypedArray([]); //=> false
+ *      _isTypedArray(null); //=> false
+ *      _isTypedArray({}); //=> false
+ */
+export default function _isTypedArray(val) {
+  var type = Object.prototype.toString.call(val);
+  return type === '[object Uint8ClampedArray]' ||
```

Comment:
I wasn't sure if this or `[...].indexOf(type)` was preferable.

Reply:
This looks fine to me